sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormRecognizerAudience.java

```diff
@@ -11,6 +11,13 @@
 /** Defines values for FormRecognizerAudience. */
 @Immutable
 public final class FormRecognizerAudience extends ExpandableStringEnum<FormRecognizerAudience> {
+
+    /**
+     * Creates a FormRecognizerAudience object.
+     */
+    public FormRecognizerAudience() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.